ci/prow/make_config.go

```diff
@@ -940,14 +1106,40 @@ func indentSection(indentation int, title string, array []string) string {
 	return indentSectionBase(indentation, title, "", array)
 }
 
+// indentMap returns the given map indented, with each key/value separated by ": "
+func indentMap(indentation int, mp map[string]string) string {
+	arr := make([]string, len(mp))
+	i := 0
+	for k, v := range mp {
+		_, err := strconv.ParseFloat(v, 64)
+		if err == nil {
```

Comment:
Factor out the common part instead of duplicating the assignment.

Reply:
Moved it to the `quote` function.